scripts/docgen/generate-docs.js

```diff
@@ -261,44 +261,70 @@ async function generateNodeSource() {
 
   // Traverse AST to get blocks annotated with @webonly and store their
   // start/end index.
-  const webOnlyBlocks = [];
-  function findWebOnlyBlocks(node) {
-    if (node.jsDoc) {
-      node.jsDoc.forEach(item => {
-        if (item.tags) {
-          item.tags.forEach(tag => {
-            if (tag.tagName.escapedText === 'webonly') {
-              webOnlyBlocks.push({ start: node.pos, end: node.end });
-              return;
-            }
-          });
-        }
-      });
+  // const webOnlyBlocks = [];
+  // function findWebOnlyBlocks(node) {
+  //   if (node.jsDoc) {
+  //     node.jsDoc.forEach(item => {
+  //       if (item.tags) {
+  //         item.tags.forEach(tag => {
+  //           if (tag.tagName.escapedText === 'webonly') {
+  //             webOnlyBlocks.push({ start: node.pos, end: node.end });
+  //             return;
+  //           }
+  //         });
+  //       }
+  //     });
+  //   }
+  //   typescript.forEachChild(node, findWebOnlyBlocks);
+  // }
+  // findWebOnlyBlocks(typescriptSourceFile);
```

Comment:
remove comments?

Reply:
Should be removed, I think this snippet is from one commit behind?